docs/deployment.md

```diff
@@ -0,0 +1,58 @@
+This repository includes code for generating a self-zip archive
```

Comment:
What does "self-zip" mean?

Reply:
Andy caught that-- I meant to write self-extracting